packages/app-cli/tests/support/pluginRepo/manifests.json

```diff
@@ -25,5 +25,43 @@
 		"_publish_hash": "sha256:e0d833b7ef1bb8f02ee4cb861ef1989621358c45a5614911071302dc0527a3b4",
 		"_publish_commit": "dev:1b5b2342fc25717b77ad9f1627c1a334e5bbae54",
 		"_npm_package_name": "@joplin/joplin-plugin-toggle-sidebars"
+	},
+	"org.joplinapp.plugins.AbcSheetMusic": {
```

Comment:
A recommended plugin is added to `manifests.json` in the test repo API (the other two plugins aren't marked as recommended). This allows testing upgrade logic related to recommended and non-recommended plugins. (See `PluginStates.test.tsx`).